news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "Beatmap Spotlights: May 2017"
+permalink: http://osunews.tumblr.com/post/162318442723/beatmap-spotlights-may-2017
+date: 2017-05-30 17:00:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/162318442723/beatmap-spotlights-may-2017
+---
+
+
+OrVid, Classic Pursuit, Chronoxia and Trigger reign supreme in the [Beatmap Spotlights](https://osu.ppy.sh/p/chart?ch=MONTH1705) of May 2017, and more to boot! Come check out what's in store this month, and maybe even win some osu!supporter for your trouble!
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+Each month, a selection of volunteers work alongside the Quality Assurance Team to pick out the best of the best for each mode. We reward the top beatmap in each category by providing **1 month of supporter status** to the mapper. **The May 2017 spotlight rankings will conclude on the 15th July 2017.**
+
+## Past Spotlights results
+
+First of all, congratulations to the winners of the April 2017 Spotlights. They receive one month of supporter status for their efforts!
+
++ **osu!:** [Toy](https://osu.ppy.sh/u/2757689)
++ **osu!taiko:** [Katdon_donKat](https://osu.ppy.sh/u/8089664)
++ **osu!catch:** [Crystal](https://osu.ppy.sh/u/1646397)
++ **osu!mania:** [Hestia-](https://osu.ppy.sh/u/393151)
+
+
+## osu!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/orvid.jpg)](https://osu.ppy.sh/s/578332)
+
+**[RLC](https://osu.ppy.sh/users/1047883)** easily took the top spot at this month's spotlight with this beautiful marathon. Reminding us of his previous masterpieces such as Anhedonia, the music offers a lot of variety and interesting rhythms over the course of more than 10 minutes. Every track of the gapless album is mapped differently, but the map still feels coherent and polished throughout. Totally worth checking out.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/get-on.jpg)](https://osu.ppy.sh/s/554084)
+
+**[sionKotori](https://osu.ppy.sh/users/1083407)** continues to impress with his newly ranked map, full of very interesting patterns which fit very well with the song and look amazing, other than giving a pretty cool sense of speed. Recommended for everyone who enjoys technical and slider-heavy maps!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/glass-no-kutsu.jpg)](https://osu.ppy.sh/s/585681)
+
+Taking a spot in almost every spotlight lately, **[Delis](https://osu.ppy.sh/users/1603923)** has made it again. This time, combining the exceptional flow and stunning variety they are known for with an emotional song by Ito Kanako. The mapping perfectly represents the impactful changes of intensity provided by the song. This map is an experience you won't regret!
```

Comment:
I don't think this is necessary to change. It's Itou or Ito, the map is written with Ito, so we stick with this (especially since the mapper is Japanese and probably got it right)